### PR TITLE
Update introductory docs

### PR DESCRIPTION
**Pull request by josevalim**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -35,13 +35,10 @@ defmodule Kino.MixProject do
 
   defp docs do
     [
-      main: "readme",
```
Comment (jonatanklosko) [inline mix.exs:38]: Will miss the logo :see_no_evil: Should we have a text-less version of the logo in the sidebar?

Comment (josevalim) [reply mix.exs:38]: Yes, let's do that instead :) Can you please do so in a separate commit?

Comment (jonatanklosko) [reply mix.exs:38]: Surething!

**Review by jonatanklosko — APPROVED** [review]

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**